docs/ktx/firestore.md

```diff
@@ -79,31 +79,21 @@ val settings = FirebaseFirestoreSettings.Builder()
                    .setPersistenceEnabled(false)
                    .build()
 
-val firestore = FirebaseFirestore.getInstance()
 firestore.setFirestoreSettings(settings)
-
-val anotherFirestore = FirebaseFirestore.getInstance(app)
-anotherFirestore.setFirestoreSettings(settings)
 ```
 
 **Kotlin + KTX**
 ```kotlin
-val firestore = Firebase.firestore {
-    setHost("10.0.2.2:8080")
-    setSslEnabled(false)
-    setPersistenceEnabled(false)
-}
-
-val anotherFirestore = Firebase.firestore(app) {
-    setHost("10.0.2.2:8080")
-    setSslEnabled(false)
-    setPersistenceEnabled(false)
+firestore.firestoreSettings = firestoreSettings {
+    host = "http://10.0.0.2:8080"
+    isSslEnabled = false
+    isPersistenceEnabled = false
 }
 
 // Alternatively
 Firebase.firestore.setSettings {
```

Comment:
Remove the alternative. Enabling the assignment `=` as the "preferred" way makes it a bit clearer that we are using the builder pattern, and that have a getter and a setter in java for `firestoreSettings`.

Reply:
@rlazo should I remove it entirely or only remove from the docs?

Reply:
Please, do remove it entirely.